Makefile

```diff
@@ -0,0 +1,12 @@
+
+LLVM_FLAGS=$(llvm-config --cxxflags)
```

Comment:
The llvm include is basically from this command.
We should just use this variable below rather than explicitly including the library paths.